root/edit/index.tt

```diff
@@ -60,18 +62,26 @@
             [% END %]
         [% ELSE %]
             <p>
-                [%- l('You must be logged in to vote on edits') -%]
-                [%- request_login() -%]
+                [%- l('You must be logged in to vote on edits.') %]
+                [% request_login() -%]
```

Comment:
Because I want this to be "You must be logged in to vote on edits. Log in", and not "You must be logged in to vote on editsLog in", as it was before.